testing/prow/gpu-operator.sh

```diff
@@ -349,12 +349,12 @@ prepare_cluster_for_gpu_operator_with_alerts() {
 
     mv ${ARTIFACT_DIR}/*__cluster__wait_for_alert* ${ARTIFACT_DIR}/alerts
 
-    if [ "${USE_PREV_RELEASE_CATALOG:-}" ]; then
+    if [ "${NFD_USE_PREV_RELEASE_CATALOG:-}" ]; then
         if oc get packagemanifests/nfd -n openshift-marketplace > /dev/null; then
             _warning  "NFD_operator_pkg_manifest" "Requested to deploy the previous catalog, but the NFD Operator packagemanifest already exists"
         fi
```

Comment:
you don't want to put a `else` here, to avoid the conflict if the package already exists?

Reply:
ack, but the failure is not deterministic, look:
![image](https://user-images.githubusercontent.com/7559202/158204536-0f559f31-e9bc-4589-a626-e6fccbbc3488.png)
I think most, if not all the failures here are due to the NFD name collision 
```
2022-03-13 23:48:34,077 p=3320 u=edge-ci-runner n=ansible | ----- FAILED ----
2022-03-13 23:48:34,077 p=3320 u=edge-ci-runner n=ansible | msg: non-zero return code
2022-03-13 23:48:34,077 p=3320 u=edge-ci-runner n=ansible | 
2022-03-13 23:48:34,078 p=3320 u=edge-ci-runner n=ansible | <command> oc get ClusterServiceVersion/nfd.4.10.0-202202241648 -oyaml -n "openshift-nfd" > /logs/artifacts/007__nfd_operator__deploy_from_operatorhub/operator_csv.yml
2022-03-13 23:48:34,078 p=3320 u=edge-ci-runner n=ansible | 
2022-03-13 23:48:34,078 p=3320 u=edge-ci-runner n=ansible | <stderr> Error from server (NotFound): clusterserviceversions.operators.coreos.com "nfd.4.10.0-202202241648" not found
2022-03-13 23:48:34,078 p=3320 u=edge-ci-runner n=ansible | ----- FAILED ----
```
that's my only explanation for this ^^^ kind of failure

Reply:
True, Something I do need to take into account. Proper error message I mean.